js-extensions/spec/ExtensionRenderer-spec.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const React = require('react');
+const jsTest = require('@jenkins-cd/js-test');
+const assert = require('chai').assert;
+const mount = require('enzyme').mount;
+const jsdom = require('jsdom');
+const ExtensionRenderer = require('../dist/ExtensionRenderer.js').ExtensionRenderer;
+const extensionStoreInstance = require('../dist/ExtensionStore.js').instance;
+
+const $ = React.createElement;
+
+function mockExtension(props) {
+    return (
+        $('h1', {}, 'Extension is a H1')
```

Comment:
Oh man ... I got so excited there. For a brief moment, I thought we were using jQuery 🚬